cellxgene_schema_cli/cellxgene_schema/utils.py

```diff
@@ -135,10 +135,12 @@ def _enforce_canonical_format(df):
         X = df.X
         if hasattr(X, "has_canonical_format") and not X.has_canonical_format:
             # this enforces canonical form; see https://docs.scipy.org/doc/scipy/tutorial/sparse.html#canonical-formats
+            logger.warning("noncanonical data found in X; converting to canonical format.")
```

Comment:
We should explicitly say we're converting to canonical by summing duplicates. In the spirit of capturing Bruce's concern that perhaps the contributor did not intend to sum the duplicate entries